server/events/pending_plan_finder.go

```diff
@@ -41,6 +43,47 @@ func (p *DefaultPendingPlanFinder) Find(pullDir string) ([]PendingPlan, error) {
 	return plans, err
 }
 
+func sortPendingPlansByModTime(repoDir, lsOut string) ([]string, error) {
+	type fileModTime struct {
+		File    string
+		ModTime time.Time
+	}
+
+	var files []fileModTime
+	for _, file := range strings.Split(lsOut, "\n") {
+		if filepath.Ext(file) != ".tfplan" {
+			continue
+		}
+		// Ignore .terragrunt-cache dirs (#487)
+		if strings.Contains(file, ".terragrunt-cache/") {
```

Comment:
Instead of ignoring this directory, can we list only the files, not directories, so we can do less filtering in lsOut